src/components/Calculator.js

```diff
@@ -1,22 +1,227 @@
+          onClick={() => {
+            this.Compute(calculatorData, '8');
+          }}
+        >
+          8
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '9');
+          }}
+        >
+          9
+        </button>
+        <button
+          type="button"
+          className="keys operators"
+          onClick={() => {
+            this.Compute(calculatorData, 'x');
+          }}
+        >
+          x
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '4');
+          }}
+        >
+          4
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '5');
+          }}
+        >
+          5
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '6');
+          }}
+        >
+          6
+        </button>
+        <button
+          type="button"
+          className="keys operators"
+          onClick={() => {
+            this.Compute(calculatorData, '-');
+          }}
+        >
+          -
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '1');
+          }}
+        >
+          1
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '2');
+          }}
+        >
+          2
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '3');
+          }}
+        >
+          3
+        </button>
+        <button
+          type="button"
+          className="keys operators"
+          onClick={() => {
+            this.Compute(calculatorData, '+');
+          }}
+        >
+          +
+        </button>
+        <button
+          type="button"
+          className="keys zero"
+          onClick={() => {
+            this.Compute(calculatorData, '0');
+          }}
+        >
+          0
+        </button>
+        <button
+          type="button"
+          className="keys"
+          onClick={() => {
+            this.Compute(calculatorData, '.');
+          }}
+        >
+          .
+        </button>
+        <button
+          type="button"
+          className="keys operators"
+          onClick={() => {
+            this.Compute(calculatorData, '=');
+          }}
+        >
+          =
+        </button>
       </div>
-    </div>
-  );
+    );
+  }
+}
+
+Calculator.propTypes = {
+  updateState: PropTypes.func.isRequired,
+  /* eslint-disable-next-line react/forbid-prop-types */
```

Comment:
- [ ] A Linter is an automated tool that runs on static code to find formatting discrepancies, non-adherence to coding standards and conventions, and find logical errors in your program. Instead of disabling `eslint`, it will be nice to fix the issue.